packages/@webex/contact-center/README.md

```diff
@@ -0,0 +1,71 @@
+### Table of Contents
+- [Getting Started](#getting-started)
+- [Developing](#developing)
+- [Building](#building) 
+- [Testing](#testing)
+- [Samples](#samples) 
+- [Consuming SDK](#consuming-sdk)
+  - [NPM](#npm)
+  - [CDN](#cdn)
+- [Kitchen Sink App](#kitchen-sink-app)
+---
+
+## Getting Started
+With the Webex Contact Center, you can effo
+
+TODO: Add the documentation links once ready
+- [Introduction to the Webex Web Calling SDK]()
+- [Quickstart guide]().
+ 
+## Developing
+
+```bash
+git clone https://github.com/\<your-fork\>/webex-js-sdk.git
+cd webex-js-sdk/
+yarn install
+```
+
+## Building
+
+If your project needs some additional steps for the developer to build the
+project after some code changes, state them here:
+
+```bash
+yarn workspaces foreach --parallel --verbose run build:src
+
+yarn build:local
+```
+
+## Testing
+
+```bash
+ yarn workspace @webex/contact-center run test
```

Comment:
We should add TODOs to all sections I think.. this command is incorrect right?

Reply:
Will fix the Readme, missed it